openwhisk/initHandler.go

```diff
@@ -0,0 +1,96 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package openwhisk
+
+import (
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+)
+
+type initRequest struct {
+	Value struct {
+		Code   string `json:",omitempty"`
+		Binary bool   `json:",omitempty"`
+	} `json:",omitempty"`
+}
+
+func sendOK(w http.ResponseWriter) {
+	// answer OK
+	w.Header().Set("Content-Type", "application/json")
+	w.Header().Set("Content-Length", "12")
+	w.Write([]byte("{\"ok\":true}\n"))
```

Comment:
if you can't remove it, then I'd rather it's tied to the byte array length rather than hardcoded to 12.

Reply:
easy fix, it was just slightly more verbose :)